src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <main class="main">
+      <form
+        data-qa="big"
+        class="main__big-search"
+      >
+        <input
+          class="big-search__input"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        data-qa="small"
+        class="main__small-search"
+      >
```

Comment:
Consider adding a `<label>` element for the input field in the form with class `main__small-search`. This will improve accessibility by providing a text label for screen readers.